routingv8/request.go

```diff
@@ -387,7 +390,7 @@ const (
 func (t *TunnelCategory) String() string {
 	switch *t {
 	case TunnelCategoryUnspecified:
-		return unspecified
+		return "None"
```

Comment:
How about defining a None const like it has been done for the other 2?

Reply:
Sounds good!